app/mailers/audit_mailer.rb

```diff
@@ -10,7 +10,7 @@ def summary(options = {})
     @audits  = Audit.authorized_as(user, :view_audit_logs, Audit).search_for(@query).limit(@limit)
 
     set_locale_for(user) do
-      mail(:to => user.mail, :subject => _("Audit summary")) do |format|
+      mail(:to => utf8_encode_email(user.mail), :subject => _("Audit summary")) do |format|
```

Comment:
Once moved, please add a bit of test coverage in test/mailers/application_mailer_test.rb to ensure it's working correctly.